docs/satellite-repos.md

```diff
@@ -18,3 +18,7 @@ We have seven satellite repos that are maintained outside of the cfgov-refresh c
 
 
 We import these satellite projects into cfgov-refresh by specifying wheel files for each in [`requirements/optional-public.txt`](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/optional-public.txt).
+
+## Local setup and usage
+
+See ["Usage"](https://cfpb.github.io/cfgov-refresh/usage/#develop-satellite-apps) for local setup and usage instructions.
```

Comment:
Instead of linking to the cfpb.github.io URL here, can we instead use a local URL like

```md
["Usage"](../usage#develop-satellite-apps)
```

so that the link works properly when testing/editing the docs locally?

We use those kind of links in a few places in these docs, for example in [this section](https://github.com/cfpb/cfgov-refresh/blob/master/docs/javascript-unit-tests.md#folder-structure-where-to-put-your-javascript-and-tests) of the JS testing doc (the "atomic components" link).